### DISCUSSION TITLE
pip install -e evennia not working

### DISCUSSION THREAD

**Discussion by Schultec** [evennia/evennia#2866] [Evennia Questions] [answered]
i am stuck at step 6 of the install guide getting the following error `(path to my project) does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.`

**Comment by audiodude** [comment, marked as answer]
In this command, `evennia` is supposed to be the path to the git clone of the evennia project. So you should have an `evennia` folder/directory with the main project source in the same directory that you're running this command.
